bundles/org.openhab.binding.wundergroundupdatereceiver/README.md

```diff
@@ -0,0 +1,88 @@
+# WundergroundUpdateReceiver Binding
+
+Many personal weather stations are only capable of submitting measurements to the wunderground.com update site.
+
+The request is in itself simple to parse, so by redirecting it to your openhab server you can intercept the values and use them to
+control items in your home. Fx. use measured wind-speed to close an awning or turn on the sprinkler system after some time without rain.
+This binding allows you to mix and match products from various manufacturers that otherwise have a closed system.
+
+It can also be used to submit the same measurements to multiple weather services in rules.
+
+## Supported Things
+
+Any weather station that sends weather measurement updates to the wunderground.com update URLs.
+It is easiest to use with stations that have a configurable target address, but can be made to
+work with any internet-connected weather station, that gets it's dns server via DHCP. 
+
+## Discovery
+
+In the initial version ther is no discovery, but it is planned for a later release. You need to manually add a thing and configure the station id.
+
+## Thing Configuration
```

Comment:
what's the URL that needs to be configured in the weatherstation?

Reply:
The binding is related to the protocol described at https://support.weather.com/s/article/PWS-Upload-Protocol?language=en_US, not any particular weather station. I think it would just cause confusion including instructions on how to configure our weather station. It may even cause users to think that the binding _only_ works with this particular weather station. On weather stations that don't allow configuring the target hostname, you would have to do other things to convince the station to submit to the openhab instance instead of wunderground.com.

Reply:
Understand. But it took me a while to figure out, how the binding is actually working. First I was thinking, that it opens it's own server and I was searching for the right port to be configured. Than I figured out, that the OH server is re-used and the OH Server Port needs to be configured.
Let me rephrase my finding. Do not describe how the station needs to be configured but how the binding is providing it's service. E.g. `http(s)://<openHAB-Server>:<openHAB-Port>/weatherstation/updateweatherstation.php`